content/models.py

```diff
@@ -0,0 +1,143 @@
+from django.db import models
+
+from user.models import Country
+
+
+class Category(models.Model):
+     name = models.CharField(max_length=45)
+     class Meta:
+         db_table = 'categories'
+
+
+class Content(models.Model):
+    title_korean = models.CharField(max_length=100)
+    main_photo   = models.CharField(max_length=1000)
+    category     = models.ForeignKey(Category, on_delete=models.CASCADE)
+    release_year = models.IntegerField(default=0000, null=True)
+    class Meta:
+        db_table = 'contents'
+
+class MovieOverview(models.Model):
+     title_original = models.CharField(max_length=100)
+     runtime        = models.IntegerField(default=0, null=True)
+     description    = models.TextField(max_length=200, null=True)
+     content        = models.ForeignKey(Content, on_delete=models.CASCADE)
+     class Meta:
+         db_table = 'movie_overviews'
+
+
+class TvProgramOverview(models.Model):
+    title_original = models.CharField(max_length=100)
+    broadcast      = models.CharField(max_length=45, null=True)
+    description    = models.TextField(null=True)
+    content        = models.ForeignKey(Content, on_delete=models.CASCADE)
+    class Meta:
+        db_table = 'tv_program_overviews'
+
+
+class SubCategory(models.Model):
+    name = models.CharField(max_length=20)
+    class Meta:
+        db_table = 'subcategories'
+
+
+class BookOverview(models.Model):
+    sub_title              = models.CharField(max_length=45)
+    sub_category           = models.ForeignKey(SubCategory, on_delete=models.CASCADE)
+    page                   = models.IntegerField(default=0, null=True)
+    book_contents          = models.TextField(max_length=1000, null=True)
+    publisher_introduction = models.TextField(max_length=1000, null=True)
+    content                = models.ForeignKey(Content, on_delete=models.CASCADE)
+    class Meta:
+        db_table = 'book_overviews'
+
+
+class ContentPhoto(models.Model):
+    content = models.ForeignKey(Content, on_delete=models.CASCADE)
+    photo   = models.CharField(max_length=1000)
```

Comment:
```suggestion
    photo_url   = models.CharField(max_length=1000)
```
url을 지정하실 때에는 이름에 url을 붙여주시는 것이 보다 직관적입니다.
Django에는 URL만을 위한 URLField가 있으니 한 번 활용해보세요 :)

Reply:
수정 후 URLField로 적용했습니다.